### ISSUE TITLE
REPRODUCTION

### ISSUE THREAD

**Issue by whuzs** [serizba/salad#2]
Hi, serizba
Thanks for the great work.

I repeated the code you provided three times, and the result is as follows. May I ask how to choose a trained model? I am currently using last.ckpt. less than 92.2
![image](https://github.com/serizba/salad/assets/38743240/9bdb942e-ee50-448b-8ace-960e9289590a)

**Comment by serizba** [comment]
Hi @whuzs 

I guess last.ckpt is a linked file to the last checkpoint, so it should be that. The small difference in performance might be explainable due to training noise. There is always a little bit of variance between different training runs, specially with an aggressive learning rate like the proposed with just 4 epochs. Also recall that although training is done on 224x224, evaluation is on 322x322. You also might get a little bit of performance gain by training and/or evaluating on full precision instead of mixed. 

You also have our trained checkpoint that we used to run our experiments.
